i2c/mcp9808_i2c/mcp9808_i2c.c

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include "pico/stdlib.h"
+#include "pico/binary_info.h"
+#include "hardware/i2c.h"
+
+/* Example code to talk to a MCP9808 ±0.5°C Digital temperature Sensor
+   
+   This reads and writes to registers on the board. 
+
+   Connections on Raspberry Pi Pico board, other boards may vary.
+
+   GPIO PICO_DEFAULT_I2C_SDA_PIN (On Pico this is 4 (pin 6)) -> SDA on MCP9808 board
+   GPIO PICO_DEFAULT_I2C_SCK_PIN (On Pico this is 5 (pin 7)) -> SCL on MCP9808 board

```

Comment:
I think the 4 and 5 here should be GP4 and GP5 ? See https://github.com/raspberrypi/pico-examples/pull/132/commits/add29c9955ff35f2f95551ea76c28f0defcdf5f0

Reply:
I'd prefer to see something like "GP4 (physical pin 6)"